.github/workflows/ci.yaml

```diff
@@ -10,8 +10,8 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  check:
-    name: Check
+  build:
+    name: Build
```

Comment:
This could be a little confusing since cargo check doesn't behave like a build ([it just compiles the packages without performing any code generation](https://doc.rust-lang.org/cargo/commands/cargo-check.html)).

Reply:
Maybe we could use `compile` and `Compile`.

Reply:
Sure, that works for me.